Packages/MIES/MIES_WaveDataFolderGetters.ipf

```diff
@@ -2455,6 +2458,18 @@ Function/Wave GetSweepSettingsKeyWave(device)
 	wv[%Units][55]     = LABNOTEBOOK_BINARY_UNIT
 	wv[%Tolerance][55] = LABNOTEBOOK_NO_TOLERANCE
 
+	wv[%Parameter][56] = "Double precision data"
+	wv[%Units][56]     = LABNOTEBOOK_BINARY_UNIT
+	wv[%Tolerance][56] = LABNOTEBOOK_NO_TOLERANCE
+
+	wv[%Parameter][57] = "Save amplifier settings"
```

Comment:
@t-b what's the motivation for documenting the `Save amplifier settings` and `require amplifier` checkboxes? Won't we know if these have been checked by the data that's present in the lab notebook?